src/bin/nydusd/fusedev.rs

```diff
@@ -113,10 +113,11 @@ impl FuseServer {
         let _ebadf = Error::from_raw_os_error(libc::EBADF);
 
         loop {
-            if let Some((reader, writer)) = self.ch.get_request().map_err(|e| {
-                warn!("get fuse request failed: {:?}", e);
-                Error::from_raw_os_error(libc::EINVAL)
-            })? {
+            if let Some((reader, writer)) = self
+                .ch
+                .get_request()
+                .map_err(|_e| Error::from_raw_os_error(libc::EINVAL))?
```

Comment:
Better not to ignore the error message here?

Reply:
Good point, better to pop the error reason up.

Reply:
Fixed